components/01-atoms/images/image/_picture.twig

```diff
@@ -0,0 +1,13 @@
+{% set picture__base_class = picture__base_class|default('picture') %}
+
```

Comment:
I'm torn on this. We definitely could, but `<picture>` is rarely used... and the actual markup will more likely come from Drupal, so we won't be defining it ourselves.

I don't think I want to add that complexity at this point (but I also don't want to remove the `<picture>` functionality, since it may be used.) So, unless you have strong feelings, I'd like to leave this part as-is for now.

Reply:
@ModulesUnraveled it isn't hurting anything by having it in but it's also not very useful if you aren't showing it. As you always say, keep your code as clean as possible. I don't have strong feelings here so it's ultimately your call.